lambdas/FetchJobsData.py

```diff
@@ -0,0 +1,62 @@
+import psycopg2
+import json
+import os
+import logging
+
+try:
+    # for local development, load environment variables from .env file
+    import dotenv
+    dotenv.load_dotenv()
+except ImportError:
+    logging.warning('dotenv not available, assuming running in AWS environment')
+
+def get_db_connection():
+    try:
+        return psycopg2.connect(
+            host=os.environ['DB_HOST'],
+            port=os.environ['DB_PORT'],
+            dbname=os.environ['DB_NAME'],
+            user=os.environ['DB_USER'],
+            password=os.environ['DB_PASSWORD'])
+    except Exception as e:
+        logging.error(f"Error connecting to the database: {e}")
+        raise
+
+def fetch_data(cursor):
```

Comment:
can you move this file or these queries into a `separate file` in `database` folder?

& we can add unit tests for them too?